mmedit/datasets/pipelines/augmentation.py

```diff
@@ -626,30 +627,51 @@ class ColorJitter:
 
     Args:
         keys (list[str]): The images to be resized.
-        to_rgb (bool): Whether to convert channels from BGR to RGB.
-            Default: False.
+        channel_order (str): Order of channel, candidates are 'bgr' and 'rgb'.
+            Default: 'rgb'.
```

Comment:
The docstring should notify users that` brightness`, `hue` etc. are also acceptable arguments. if `kwargs` is directly passed to `torchvision.trasnforms.ColorJitter`, we can simply tell users to follow args of `torchvision.transforms.ColorJitter`. And for the same reason, `__repr__` usually contain full kwargs, so the expression returned by `__repr__` constructs the same transformation object.